go/chat/unfurl/utils.go

```diff
@@ -9,10 +9,20 @@ import (
 	"golang.org/x/net/publicsuffix"
 )
 
-func GetHostname(uri string) (res string, err error) {
+func GetDefaultFaviconUrl(uri string) (string, error) {
 	parsed, err := url.Parse(uri)
 	if err != nil {
-		return res, err
+		return "", err
+	}
+	parsed.Path = "favicon.ico"
```

Comment:
this is the default for most sites (keybase doesn't drop a tag on the page but has this path)